src/test/kotlin/org/rust/ide/typing/paste/RsConvertJsonToStructCopyPasteTest.kt

```diff
@@ -396,6 +396,11 @@ class RsConvertJsonToStructCopyPasteTest : RsTestBase() {
         }
     """, """{"a": [1, 2, 3,],}""")
 
+    override fun setUp() {
+        super.setUp()
+        CONVERT_JSON_ON_PASTE.setValue(true, testRootDisposable)
```

Comment:
There is also a second file with toolchain tests, maybe that one will need this change too?

Reply:
You mean `RsConvertJsonToStructCopyPasteToolchainTest` from #8596, right? Yeah, it will need this too (it's not merged yet)